drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -249,6 +372,22 @@ static int gpio_ite_configure(const struct device *dev,
 	else
 		*reg_gpotr &= ~mask;
 
+	/* 1.8V or 3.3V */
```

Comment:
assign the values to reg_1p8v/mask_1p8v here.

Reply:
Done.